arwen/contexts/runtime.go

```diff
@@ -531,7 +531,7 @@ func (context *runtimeContext) FailExecution(err error) {
 	context.host.Output().SetReturnMessage(message)
 	context.SetRuntimeBreakpointValue(arwen.BreakpointExecutionFailed)
 
-	logRuntime.Trace("execution failed", "message", message)
+	logRuntime.Trace("execution failed", "message", err.Error())
```

Comment:
this can panic

Reply:
better add an if before trace. this call is not really called with err = nil. but it is better to be safe